MM-12290 inline image collapse (#4495)
mattermost/docs: source/help/settings/account-settings.rst

When available, the first web link in a message will show a preview of the website content below the message. This `setting must be enabled by your System Admin <https://docs.mattermost.com/administration/config-settings.html#link-previews>`__.

Default Appearance of Image Previews
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> When messages in Mattermost include images, an image link preview can display directly below the message for image attachements as well as `in-line images <https://docs.mattermost.com/help/messaging/formatting-text.html#in-line-images.html>`__ over 100px in height.

Author: ```suggestion
When messages in Mattermost include images, an image preview can display directly below the message for image attachments, image link previews, and `in-line images <https://docs.mattermost.com/help/messaging/formatting-text.html#in-line-images.html>`__ over 100px in height.
```

> Enable this preference to show image previews. Disable this preference to hide link previews in Mattermost.

Reviewer: enabling/disabling just sets the default expanded/collapsed states, so technically not "hiding"
